scene/2d/joint_2d.cpp

```diff
@@ -289,11 +292,56 @@ real_t PinJoint2D::get_softness() const {
 	return softness;
 }
 
+void PinJoint2D::set_angular_limit_lower(real_t p_angular_limit_lower) {
+	angular_limit_lower = p_angular_limit_lower;
```

Comment:
```suggestion
	if (angular_limit_lower == p_angular_limit_lower) {
		return;
	}
	angular_limit_lower = p_angular_limit_lower;
```
Avoid changes when value doesn't change